.eslintrc.js

```diff
@@ -1,51 +1,41 @@
 module.exports = {
-	"env": {
-		"commonjs": true,
-		"es6": true,
-		"node": true
-	},
-	"extends": "eslint:recommended",
-	"overrides": [
-		{
-			"files": ["**/*.ts"],
-			"parser": "@typescript-eslint/parser",
-			"extends": [
-				"eslint:recommended",
-				"plugin:@typescript-eslint/eslint-recommended",
-				"plugin:@typescript-eslint/recommended"
-			],
-			"plugins": ["@typescript-eslint"],
-			"rules": {
-				"@typescript-eslint/camelcase": "off",
-				"@typescript-eslint/no-this-alias": "off",
-				"@typescript-eslint/no-var-requires": "off",
-				"@typescript-eslint/no-explicit-any": "off"
-			}
-		}
-	],
-	"globals": {
-		"Atomics": "readonly",
-		"SharedArrayBuffer": "readonly"
-	},
-	"parserOptions": {
-		"ecmaVersion": 2018
-	},
-	"rules": {
-		"indent": [
-			"error",
-			"tab"
-		],
-		"linebreak-style": [
-			"error",
-			"unix"
-		],
-		"quotes": [
-			"error",
-			"double"
-		],
-		"semi": [
-			"error",
-			"always"
-		]
-	}
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    sourceType: "module",
+  },
+  env: {
+    commonjs: true,
+    es6: true,
+    node: true,
+  },
+  plugins: ["@typescript-eslint/eslint-plugin"],
+  extends: [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended",
+  ],
+  rules: {
+    "@typescript-eslint/interface-name-prefix": "off",
+    "@typescript-eslint/explicit-function-return-type": "off",
+    "@typescript-eslint/no-explicit-any": "off",
+    "@typescript-eslint/no-unused-vars": "off",
+    "@typescript-eslint/camelcase": "off",
+    "@typescript-eslint/no-this-alias": "off",
+    "@typescript-eslint/no-var-requires": "off",
+    "@typescript-eslint/no-explicit-any": "off",
+    "object-curly-spacing": ["error", "never"],
+    "space-before-function-paren": ["error", "never"],
```

Comment:
I think this should be always.

```suggestion
    "space-before-function-paren": ["error", "always"],
```